.travis.yml

```diff
@@ -30,3 +30,15 @@ install:
 script:
     - phpunit --exclude-group infra
     - if [[ "$CHECK_PHP_SYNTAX" == "yes" ]]; then vendor/bin/php-cs-fixer fix --config=.php_cs --dry-run --no-interaction --diff; fi;
+
+You need to configure your Travis job to send Webhook notifications.
+
+Add the following to your .travis.yml configuration:
```

Comment:
Travis is not fine with the two lines above :trollface: 

> ERROR: An error occured while trying to parse your .travis.yml file.
Please make sure that the file is valid YAML.
http://lint.travis-ci.org can check your .travis.yml.